src/bucket.ts

```diff
@@ -765,4 +765,23 @@ export default class Bucket {
       aggregate: !!options.aggregate,
     });
   }
+
+  /**
+   * Retrieves bucket server endpoints.
+   *
+   * @return {Object}
+   */
+  get endpoint(): Record<string, Function> {
+    return {
+      toString: () => endpoint.bucket(this.name),
+      bucket: () => endpoint.bucket(this.name),
```

Comment:
I think we can remove these two since it would make more sense to have a property directly on `Bucket` representing the URL. I'm not sure what that property should be named, but we can open another issue to discuss that.